index.js

```diff
@@ -142,7 +183,12 @@ const line = []
 
 // *** Uncomment the lines below to test
 // console.log("%cScope & Closures - Question 1", "color: red")
+let ticketNumber = 1
 
+function takeATicketNumber(){
```

Comment:
Cool! Great use of the global scope. You could also start `ticketNumber` at 0 and increment before returning their line number. Same difference.